src/vm/proftoeeinterfaceimpl.cpp

```diff
@@ -2595,21 +2599,16 @@ HRESULT ProfToEEInterfaceImpl::GetCodeInfo3(FunctionID functionId,
             CodeVersionManager* pCodeVersionManager = pMethodDesc->GetCodeVersionManager();
             ILCodeVersion ilCodeVersion = pCodeVersionManager->GetILCodeVersion(pMethodDesc, reJitId);
 
-            // Now that tiered compilation can create more than one jitted code version for the same rejit id
-            // we are arbitrarily choosing the first one to return. To return all of them we'd presumably need
-            // a new profiler API.
```

Comment:
This comment is still correct - we are arbitrarily choosing an implementation. The comment could also mention the new APIs you've added rather than refer to hypothetical new APIs.

Reply:
Done